### DISCUSSION TITLE
Update be/fe.conf file Does restart affect fe/be node?

### DISCUSSION THREAD

**Discussion by dzygoon** [apache/doris#29899] [A - General / Q&A]
My cluster has 4 fe node and 3 be node.Now,I want to update be/fe.conf some item,for example "JAVA_OPTS",Should i restart the fe/be node?
If i restart.I wonder what the impact is.
thanks.

**Comment by dzygoon** [comment]
On my cluster,I update FE conf after,stop node and restart node.BE node has same operation.When done they work normally.
Because my cluster dont have anydata,They were not officially useing.So i can do this.
But i want to know,If my cluster was `prod` cluster.how should i do?

**Comment by zy-kkk** [comment]
https://doris.apache.org/docs/admin-manual/config/fe-config/#set-configuration-items
https://doris.apache.org/docs/admin-manual/config/be-config#set-configuration-items

**Reply by dzygoon** [reply]
yep.I know that.
But I wonder if restarting the FE/BE node will have any impact?For example,FE metadata lost or BE data cant be recover.Especially if my cluster is in use. @zy-kkk
